dbt/loader.py

```diff
@@ -17,7 +17,9 @@ def load_all(cls, root_project, all_projects):
         for loader in cls._LOADERS:
             nodes.update(loader.load_all(root_project, all_projects, macros))
 
-        return ParsedManifest(nodes=nodes, macros=macros)
+        manifest = ParsedManifest(nodes=nodes, macros=macros)
```

Comment:
tell the ParsedManifest to process itself:
 - process refs
 - incorporate Schema Spec info


UnparsedManifest.parse() ---> ParsedManifest

Reply:
i've started an issue for this here: https://github.com/fishtown-analytics/dbt/issues/821